tests/test_routes.py

```diff
@@ -9,48 +9,34 @@
 from service.common import status
 from service.models import db, PaymentMethod
 
+
 DATABASE_URI = os.getenv(
     "DATABASE_URI", "postgresql+psycopg://postgres:postgres@localhost:5432/postgres"
 )
-
-
-######################################################################
-#  T E S T   C A S E S
-######################################################################
-# pylint: disable=too-many-public-methods
-class TestPaymentsService(TestCase):
-    """REST API Server Tests"""
-
-    @classmethod
-    def setUpClass(cls):
-        """Run once before all tests"""
-        app.config["TESTING"] = True
-        app.config["DEBUG"] = False
-        # Set up the test database
-        app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
-        app.logger.setLevel(logging.CRITICAL)
-        app.app_context().push()
-
-    @classmethod
-    def tearDownClass(cls):
-        """Run once after all tests"""
-        db.session.close()
-
-    def setUp(self):
-        """Runs before each test"""
-        self.client = app.test_client()
-        db.session.query(PaymentMethod).delete()  # clean up the last tests
-        db.session.commit()
-
-    def tearDown(self):
-        """This runs after each test"""
-        db.session.remove()
+BASE_URL = "/payments"
 
     ######################################################################
     #  P L A C E   T E S T   C A S E S   H E R E
     ######################################################################
 
-    def test_index(self):
+    
+
+    def test_update_payment(self):
+        """It should Update an existing Payment"""
+        # create a payment to update
+        test_payment = PaymentFactory()
+        response = self.client.post(BASE_URL, json=test_payment.serialize())
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # update the payment
+        new_payment = response.get_json()
+        logging.debug(new_payment)
+        new_payment["name"] = test_payment.name
+        response = self.client.put(f"{BASE_URL}/{new_payment['id']}", json=new_payment)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        updated_payment = response.get_json()
+        self.assertEqual(updated_payment["name"], test_payment.name)
+        
         """It should call the home page and receive information about existing methods"""
```

Comment:
these should be separate test cases. it seems you removed part of the `index` test case and then merged the rest of it with your test case, even though they test different thing